### PR TITLE
fix: load readme as long description in setup.py

### PR DESCRIPTION
**Pull request by mcembalest**
> [!IMPORTANT]
> Load `long_description` from `README.md` and bump version to 3.3.1 in `setup.py`.
> 
>   - **Setup Configuration**:
>     - Load `long_description` from `README.md` in `setup.py` instead of using `description`.
>   - **Version Update**:
>     - Bump version from `3.3.0` to `3.3.1` in `setup.py`.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=nomic-ai%2Fnomic&utm_source=github&utm_medium=referral)<sup> for 78c350044975d20cab156d6b5dbf828da3117f5d. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,12 +6,15 @@
 
 description = "The official Nomic python client."
 
+with open('README.md') as f:
+    long_description = f.read()
+
 setup(
     name="nomic",
-    version="3.3.0",
```
Comment (zanussbaum) [inline setup.py:11]: You don't need to bump the version, release please will handle it

**Review by zanussbaum — DISMISSED** [review]

**Review by bmschmidt — APPROVED** [review]
LGTM

File: setup.py (hunk 1 above)
Comment (bmschmidt) [reply setup.py:11]: Oh yeah +1

**Comment by bmschmidt** [comment]
Noting @mcembalest I added fix to the PR title here, which is necessary to make it actually trigger the release please note

**Pull request merged**